packages/connector-alipay/src/index.ts

```diff
@@ -104,7 +104,7 @@ export class AlipayConnector implements SocialConnector {
   };
 
   public getUserInfo: GetUserInfo = async (accessTokenObject) => {
-    const config = await this.getConfig(this.metadata.id);
+    const config = await this.getConfig(this.metadata.target, this.metadata.platform);
```

Comment:
Can we pass in "id"(in db) to this instance? Can create an issue to track.

Reply:
LOG-2397 was added to track this.